test/swaggers/specification/scenarios/resource-manager/Microsoft.Test/2016-01-01/test.json

```diff
@@ -0,0 +1,224 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "StorageManagementClient",
+    "description": "The Storage Management Client.",
+    "version": "2016-01-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Test/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_CheckNameAvailability",
+        "description": "Checks that the storage account name is valid and is not already in use.",
+        "x-ms-examples": {
+          "storageAccountCheckNameAvailability": {"$ref": "./examples/storageAccountCheckNameAvailability.json"}
```

Comment:
you could probably have multiple examples for 2 operations (one for the forward slashes, one of the "?"), instead of multiple operations with one example each, it may simplify test.json.

Reply:
I wanted to provide a scenario that has both the issues and verify whether our tool correctly catches it.